src/assessment.js

```diff
@@ -39,14 +46,14 @@ var q2 = [/* your answer here */];
 //
 //     `['alderaan', 'chartreuse', 'frickinLasers', 'laserSize']`
 //
-var q3 = [/* your answer here */];
+var q3 = ['laserSize'];
 //
 
 // Edit line 6 to read `conosole.log(this)`. Insert a line between 7 and 8 that
 // invokes `fire`. If we invoked `deathStar` in a browser environment, what
 // would the execution context of line 6 be?
 //
-var q4 = "/* your answer here */";
+var q4 = "deathStar.call(fire)"; //if we invoked deathStar in a brower environment, the context would be the window
```

Comment:
No, sorry - `.call` is a method on functions. Additionally, you are incorrect about the context - it would not be `window`.